cli/mfg.c

```diff
@@ -182,7 +182,14 @@ static void print_security_config(struct switchtec_security_cfg_state *state,
 	printf("\tI2C Recovery TMO: \t\t%d Second(s)\n",
 		state->i2c_recovery_tmo);
 	printf("\tI2C Port: \t\t\t%d\n", state->i2c_port);
-	printf("\tI2C Address (7-bits): \t\t0x%02x\n", state->i2c_addr);
+	if (state->gen == SWITCHTEC_GEN5) {
+		printf("\tI2C Address (7-bits): \t\t0x%02x\n",
+		       state->i2c_addr>>1);
```

Comment:
Why not use 7-bit value when we assign it to state->i2c_addr?

Reply:
Yeah, this seems highly questionable.... We should store the address the same for each generation -- not have a change in how it's printed...